src/SentenceStudio/Pages/VocabularyMatching/VocabularyMatchingPage.cs

```diff
@@ -503,7 +503,7 @@ async Task RecordMatchActivity(bool isCorrect)
         {
             var activity = new UserActivity
             {
-                Activity = "VocabularyMatching",
+                Activity = SentenceStudio.Shared.Models.Activity.VocabularyMatching.ToString(),
                 Input = "match_attempt",
```

Comment:
let's record the target language word instead of "match_attempt" so we know what words the user struggles with and vice versa